service/src/main/java/uk/gov/hmcts/reform/fnp/model/fee/FeeType.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.fnp.model.fee;
+
+import com.google.common.collect.ImmutableList;
+import uk.gov.hmcts.reform.fpl.enums.OrderType;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toUnmodifiableList;
+
+public enum FeeType {
+    C2_WITHOUT_NOTICE,
+    C2_WITH_NOTICE,
+    CARE_ORDER,
+    EDUCATION_SUPERVISION_ORDER,
+    EMERGENCY_PROTECTION_ORDER,
+    INTERIM_CARE_ORDER,
+    INTERIM_SUPERVISION_ORDER,
+    OTHER,
+    PLACEMENT,
+    SUPERVISION_ORDER;
+
+    public static List<FeeType> fromOrderType(List<OrderType> orderTypes) {
+        if (orderTypes == null) {
+            return ImmutableList.of();
+        }
+
+        return orderTypes.stream()
+            .map(FeeType::fromOrderType)
+            .collect(toUnmodifiableList());
+    }
+
+    public static FeeType fromOrderType(OrderType orderType) {
```

Comment:
Could we have unit test iterating on all values of OrderType to ensure we added mapping him? I prefer failure if someone forgets about something than IllegalStateException later in the runtime.